editor/editor_node.cpp

```diff
@@ -3710,7 +3710,9 @@ void EditorNode::set_addon_plugin_enabled(const String &p_addon, bool p_enabled,
 	// Only try to load the script if it has a name. Else, the plugin has no init script.
 	if (script_path.length() > 0) {
 		script_path = addon_path.get_base_dir().path_join(script_path);
-		scr = ResourceLoader::load(script_path, "Script", ResourceFormatLoader::CACHE_MODE_IGNORE);
+		// We should use the cached version on startup to prevent a script reload
+		// if it is already loaded and potentially running from autoloads.
```

Comment:
```suggestion
		// We should use the cached version on startup to prevent a script reload
		// if it is already loaded and potentially running from autoloads. See godotengine/godot#100750
```
I don't think the full context of the issue is fully understandable from just the comment.

Reply:
While we're here, I've been told the appropriate issue reference format would be GH-100750 :)

Reply:
Good to know, never knew we had guidelines for this. Let's hope no one ever wants to link a proposal :sweat_smile:

Reply:
Same here! Did not know that! Thanks!
And my comment was wrong... it should have been: `// We should not use the cached versionn on startup...`